examples/http/onvif_camera.js

```diff
@@ -0,0 +1,324 @@
+/**
+ * Domotz Custom Driver 
+ * Name: ONVIF camera monitoring
+ * Description: Monitors the video capturing status of an ONVIF camera to check if capturing is still working or has stopped.
+ * 
+ * Communication protocol is ONVIF
+ * 
+ * Tested on LILIN camera model LD2222
+ *
+ * Creates a Custom Driver table  with video capturing status for each camera profiles
+ * 
+ **/
+
+// Device credentials
+var username = D.device.username();
+var password = D.device.password();
+
+// Table for Streaming status
+var table = D.createTable(
+    "Video Streaming Status",
+    [
+        { label: "Streaming status" }
+    ]
+);
+
+// HTTP request parameters
+var url = "/onvif/device_service"; 
+
+// This parameter specifies which camera profile's video capturing status to monitor
+// Example usage:
+// profileName = ["Profile1", "Profile2"] to monitor specific profiles
+// or
+// profileName = ["All"] to monitor all profiles.
+var profileName = D.getParameter("profileName");
```

Comment:
this parameter filtering does not work as you have suggested on the example usage

you are actually doing filtering on profile tokens and not the names
(it should be the names)